test/hotspot/jtreg/serviceability/jvmti/VMObjectAlloc/libVMObjectAlloc.cpp

```diff
@@ -48,6 +59,13 @@ VMObjectAlloc(jvmtiEnv *jvmti,
   }
 }
 
+static void JNICALL
+VMDeath(jvmtiEnv *jvmti, JNIEnv* jni) {
+  RawMonitorLocker locker(jvmti, jni, event_mon);
+
+  printf("VMDeath\n");
```

Comment:
We shouldn't use printf in tests. It doesn't flush stdout and output might be missed. So it is always unclear if we had vmdeath or not. Please use LOG always.

Reply:
Fixed (also replaced pre-existing printf with LOG in the test)